spec/cache/senders/impressions_formatter_spec.rb

```diff
@@ -68,14 +69,15 @@
                                  b: 'foo2',
                                  r: 'custom_label2',
                                  c: 123_499,
-                                 pt: nil }]
+                                pt: nil,
+                                properties: "null" }]
```

Comment:
if there are no properties, shouldn't it be `properties: nil`? and so, no `properties` field is sent in the `POST /testImpressions/bulk` payload.

Reply:
Correct, will fix it.